packages/plugin/automatic-persisted-queries/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@graphql-ez/automatic-persisted-queries",
+  "version": "1.0.0",
+  "homepage": "https://www.graphql-ez.com",
+  "author": "Clayton Collie <collie.clayton@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PabloSzx/graphql-ez",
+    "directory": "packages/plugin/automatic-persisted-queries"
+  },
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
+    },
+    "./*": {
+      "require": "./dist/*.js",
+      "import": "./dist/*.mjs"
+    }
+  },
+  "typings": "dist/index.d.ts",
+  "typescript": {
+    "definition": "dist/index.d.ts"
+  },
+  "scripts": {
+    "dev": "bob-esbuild watch",
+    "prepack": "bob-esbuild build",
+    "postpublish": "gh-release",
+    "test": "jest"
+  },
+  "dependencies": {
+    "tiny-lru": "^7.0.6",
+    "@envelop/core": "^1.0.3",
+    "@envelop/types": "^1.0.2",
```

Comment:
You should reuse the Envelop returned from graphql-ez, graphql-ez re-exports everything from Envelop

Reply:
this ensures that graphql-ez is always 100% aligned with Envelop